### PR TITLE
[JInstaller::parseXMLInstallFile] Parse also metadata section of manifest files

### PR DESCRIPTION
**Pull request by andrepereiradasilva**
### Summary of Changes

This PR allow JInstaller::parseXMLInstallFile to also parse the metadata section in xml files (ex: xml language files).
### Testing Instructions

Code review.
- add this to your isis/index.php and refresh the page you will see the array with that xml file.

``` php
print_r(JInstaller::parseXMLInstallFile(JPATH_SITE . '/language/en-GB/install.xml'));
```
- Now add this to the end of the en-GB `install.xml` file:

``` xml
    [...]
    </files>
    <metadata>
        <name>English (en-GB)</name>
        <tag>en-GB</tag>
        <rtl>0</rtl>
        <locale>en_GB.utf8, en_GB.UTF-8, en_GB, eng_GB, en, english, english-uk, uk, gbr, britain, england, great britain, uk, united kingdom, united-kingdom</locale>
        <firstDay>0</firstDay>
        <weekEnd>0,6</weekEnd>
    </metadata>
    <params />
</extension>
```
- Refresh page.  You will see the metadata section IS NOT there.
- Apply patch.
- Refresh page.  You will see the metadata section IS there.
- Install any extension, language to confirm all installs fine.
### Documentation Changes Required

None.

### CODE REVIEW

File: libraries/cms/installer/installer.php [hunk 1]
```diff
@@ -2226,6 +2226,16 @@ public static function parseXMLInstallFile($path)
 			}
 		}
 
+		// If we have metadata (ex: language xml files)
+		if ($xml->metadata && count($xml->metadata->children()))
+		{
+			$data['metadata'] = array();
+			foreach ($xml->metadata->children() as $xmlTag)
```
Comment (zero-24) [inline libraries/cms/installer/installer.php]: can we have a clean line before the `foreach`?

**Review by zero-24 — CHANGES_REQUESTED** [review]

**Comment by zero-24** [comment]
just one smal one ;)

**Review by zero-24 — APPROVED** [review]

**Comment by zero-24** [comment]
i'm not sure. But the code gives me the same with and without the patch and anytime without the metadata? I'm confused but maybe i miss something?

**Comment by zero-24** [comment]
I have tested this item :white_check_mark: successfully on 3b810e0950373ba50d745586cb2d89780529e004<br /><br />works. it was a wrong setup problem here ;) Thanks<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/12081">issues.joomla.org/joomla-cms/12081</a>.</sub>

**Comment by mbabker** [comment]
This PR needs a new owner or it will have to be closed as abandoned.

**Comment by zero-24** [comment]
The bot is was your friend this day @mbabker ;)

**Comment by franz-wohlkoenig** [comment]
If this PR get no Response, it will be closed at 23th July 2017.

**Comment by ghost** [comment]
closed as stated above.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/12081">issues.joomla.org/tracker/joomla-cms/12081</a>.</sub>

**Pull request closed without merging**